tinygrad/nn/state.py

```diff
@@ -226,3 +226,69 @@ def persistent_load(self, pid): return deserialized_objects.get(pid, pid)
         base_offset += 8 + lens[i]
       f.seek(rwd)
       return TorchPickle(f).load()
+
+class GGUFConv:
```

Comment:
Converter is a better name I think, Conv is convolution

Reply:
I agree! I was unhappy with this class definition for a couple of other reasons, which is why I have moved the conversion logic to a function `ggml_data_to_tensor`, which also saved 7 sz.py lines.